test/compress/harmony.js

```diff
@@ -46,6 +46,24 @@ typeof_arrow_functions: {
     expect_exact: "\"function\";"
 }
 
+template_strings: {
+    input: {
+        ``;
+        `xx\`x`;
+        `${ foo + 2 }`;
+        ` foo ${ bar + `baz ${ qux }` }`;
+    }
+    expect_exact: "``;`xx\\`x`;`${foo+2}`;` foo ${bar+`baz ${qux}`}`;";
```

Comment:
It also tests that the stuff inside `${}` got compressed and as such not mistaken for the outside string.